### ISSUE TITLE
Fedidcard Stage 2 (forward action)

### ISSUE THREAD

**Issue by anatecture** [GSA/EDX#135]
tl/dr on the fedidcard meeting: 

1. They'd like to not go with a drupal migration; willing to spend their time and resources in moving somewhere else
2. They'd like to maintain a redirect for the fedidcard.gov url
3. They don't have any customer data beyond google analytics
5. They have a substantial daily burden of manually uploading some data...concerned that that burden would increase if they have to work through another team to make those changes. 

Next steps: 

1. They're willing to do a small hcd project (~4 weeks) to learn about how customers use the site. Sheila D is working to see if they can get on the agenda for a monthly meeting of their customers to talk, next one is in January. 
2. Ben is reaching out to gsa IT to see if they can get an estimate for a migration project, dollars and timeline
3. We need to talk to Sarah B to see _if_ gsa.gov can support this work, both immediate/migration-related and ongoing/program-related
4.  map with the team a continuity plan so that the migration of this website and its features is painless / invisible to customers.
5. work with team and OSC on content to complete migration

_Originally posted by @anatecture in https://github.com/GSA/EDX/issues/126#issuecomment-990142900_

**Comment by anatecture** [comment]
https://github.com/GSA/EDX/issues/126#issuecomment-990144436

**Comment by anatecture** [comment]
[Pulling my own comment into this new issue](https://github.com/GSA/EDX/issues/126#issuecomment-991197784)

`That would be very helpful @ryanwoldatwork - I'm wondering the same thing: why is this a recurring burden?`

`My root level question is: why do they need to update this list of open/closed locations every day? Do customers actually use the list? Is the list required my law or policy? But in the interest of meeting their immediate need to fulfill their mission, I think we leave those questions aside and figure out if there's a possibility for direct-from-data-source integration.`

**Comment by mvogelgesang** [comment]
Some considerations for fedidcard.gov...

The existing webform may not be supported on gsa.gov as a means for protecting the server from spam. However, embedding a form from another service like Qualtrics, Medallia, or Salesforce could do the trick. At the end of the day, folks need to be able to reach out to the right team.

"Find a Credentialing Center" would require a lift and shift. However, there is an opportunity to make the experience much better. The team could leverage structured data on the page so that each credentialing center would appear as a business in Google or Bing search along with hours,  and perhaps specific directions.

Search - There is a lot of specific content on this site that may get muddled with other content on gsa.gov with a similar name. If there was a way to give users the ability to just search within the fedidcard silo, that would be a big boost.

**Comment by anatecture** [comment]
Circled with Liz C. to discuss today. Here's what came out: 

For CY22: fedidcard is going to solve two problems this year (1) migrate to gsa.gov as entirely as possible and pull down this stand alone site and (2) go through training to learn more about customer research and agile practices. 

Liz is going to work on the budget and scope, and I'm reserving a slot for one of their team members in the EDX + Lab digital literacy training in April-May. 

While migrating, we'll work with them to solve one additional problem, if possible, and that is the daily, manual upload of the open and closed sites. 

Full scope of work to be sussed out in January.

**Comment by anatecture** [comment]
Bringing two forward action points from [original issue](https://github.com/GSA/EDX/issues/126) up here: 

- [ ] map with the team a continuity plan so that the migration of this website and its features is painless / invisible to customers. 
- [ ] work with team and OSC on content to complete migration

**Comment by anatecture** [comment]
emailed Liz C. to touch base 01/06/2022

**Comment by anatecture** [comment]
Update from OSC's SB + my reply. Checking off point (2) in comment above, as this starts the process outlined in the point as written is poorly worded, since it has no end state. 

via email, from Sarah 01/14: 
The Issue

- The program has two public facing sites, fedidcard.gov site and the USAccess scheduling site. The predominant reason visitors go to these sites is to book an appointment at a USAccess center.

- Thousands of visitors look-up USAccess center locations on fedidcard.gov, but then must look-up locations again to make an appointment USAccess. It appears the reason for the two sets of look-ups is because customers of the USAccess program have the option to use the USAccess scheduling system. If customers use a different scheduling system, they often "opt" out of being listed on USAccess but can still opt to be listed on fedidcard.gov.  

- This setup has resulted in a large amount of duplication, as most sites do appear to be listed in both locations. It would seem easy enough to indicate on the USAccess scheduling site if a person needs to use a different scheduling system. That change would allow GSA to retire the lookup feature for fedidcard.gov, save time and eliminate confusion for customers, and reduce IT maintenance costs long term.

My reply: 
- Summary of prior actions and [link to issue.](https://github.com/GSA/EDX/issues/126) 
- scheduled a meeting w/ @anatecture @mvogelgesang @AKM399 @RachelF and SB to discuss how to proceed.

**Comment by anatecture** [comment]
01/18: @mvogelgesang @anatecture @AKM399 @anatecture met w/ Sarah B. and Patrick S. to talk about where project is. gsa.gov and fedidcard team are working well to shift content from fedidcard over to gsa.gov. More work needs to be done on the features side. Actions: 

1. EDX (@AKM399) to research  usaacess team
2. After research, EDX will reach out to usaaccess team to schedule a first meeting. Goal: to start unraveling their cx path. Reference: point (2) below for OSC's read on the situation.  

Conclusions from meeting: 

1. gsa.gov _could_ host a mapping feature and does want to, but doesn't need to at this time
2. More important, in Sarah B's eyes, to get with usaaccess team and figure out if we can build out the cx on usaaccess to include an end-to-end booking experience including: 
    1. finding a location
    2. scheduling an appointment
    3. mapping the route to that location 
3. Additionally, Sarah B. would like to see if the following features could be integrated into usaacess: 
    1. directions
    2. text field of special instructions
